core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -1820,6 +1820,9 @@ public boolean isTransactionRelevant(Transaction tx) throws ScriptException {
         // Now for each pending transaction, see if it shares any outpoints with this tx.
         Set<Transaction> doubleSpendTxns = Sets.newHashSet();
         for (Transaction p : candidates.values()) {
+            if(p.getHash().equals(tx.getHash())) {
```

Comment:
If the check for double spends is removed, this would also be uneccessary, should I leave this or not?